.circleci/deployment/workflows.yml

```diff
@@ -51,7 +54,10 @@
                 - develop
 
   production-deployment:
-    unless: << pipeline.parameters.run_dev_deployment >>
+    unless:
+      or:
+        - << pipeline.parameters.run_dev_deployment >>
+        - << pipeline.parameters.run_nightly_owasp_scan >>
     jobs:
```

Comment:
@ADPennington added this logic in for production-deployment now too. Good catch!